buildkite/src/Jobs/Release/MinaArtifactOnlyDebianBullseye.dhall

```diff
@@ -0,0 +1,28 @@
+let ArtifactPipelines = ../../Command/MinaArtifact.dhall
```

Comment:
What's difference between this artifact and others?

Reply:
This is usability job which one can use to quickly build debian by using `!ci-debian-me`.